mu/sims/sub_sims/mcp3427.py

```diff
@@ -0,0 +1,25 @@
+from mu.harness.project import StoreUpdate
+from mu.harness.sub_sim import SubSim
+from mu.protogen import stores_pb2
+from mu.protogen import mcp3427_pb2
+
+MCP3427_KEY = (stores_pb2.MuStoreType.MCP3427, 0)
+NUM_MCP3427_CHANNELS = 2
+
+class Mcp3427(SubSim):
+    # val1 writes channel1 of mcp3427, val2 channel2
+    def update_mcp3427(self, val1, val2, fault_flag):
+        mcp3427_msg = mcp3427_pb2.MuMcp3427Store()
+        mcp3427_msg.readings.extend([0] * NUM_MCP3427_CHANNELS)
```

Comment:
I feel like there's maybe another way to do this as mcp23008 also has a repeated value, but doesn't use it?
I get a keyerror without it

Reply:
Just tested it, turns out it's actually a bug in the mcp23008 sim since the test didn't run the update_pin function. This way is correct.